src/components/NormalizationDashboard.jsx

```diff
@@ -5,16 +5,18 @@ import WriterDashBoardTabs from "./WriterDashBoardTabs.jsx";
 
 function NormalizationDashboard() {
   return (
-    <div className="bg-bg">
-      <div className="m-2">
+    <div className="bg-grey-40 h-screen">
+      <div className="mx-[139px]">
+      <div>
```

Comment:
Remove all unnecessary divs.

Reply:
This is necessary

Reply:
Necessary how ?

Reply:
we need bg color for full screen but spacing only for content if i move it to single div the bg color will be only for the conent